### ISSUE TITLE
Disabling values reporting breaks bug reporting

### ISSUE THREAD

**Issue by keithhand** [kubecost/cost-analyzer-helm-chart#1826]
**Describe the bug**  
When disabling the value `.reporting.valuesReporting` you can no longer generate a bug report.

**To Reproduce**  
1. Install Kubecost with `--set reporting.valuesReporting=false`
2. Navigate to the setting page and try to generate a bug report
3. See the following console error:
```
Uncaught (in promise) SyntaxError: Unexpected token 'V', "Values rep"... is not valid JSON
```

**Expected behavior**  
Bug reports should generate with or without values reporting.

**Comment by AjayTripathy** [comment]
Agree this looks like a bug. Tagging as 1.99 for triage.

**Comment by AjayTripathy** [comment]
Let's do this in v1.100

**Comment by AjayTripathy** [comment]
@nealormsbee feels like a frontend issue? Should be a quick one in v1.100

**Comment by nealormsbee** [comment]
Addressed in our FE codebase! This will be fixed in v1.100.
